<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>win_change_color</class>
 <widget class="QWidget" name="win_change_color">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Изменение цвета текста истории</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ТЕКСТ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Принять</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>191</width>
     <height>212</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbl_color">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Цвета</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbt_black">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Чёрный</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbt_red">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Красный</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbt_blue">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Синий</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbt_orange">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Оранжевый</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbt_green">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Зелёный</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
